cumulative/cumulative.go

```diff
@@ -60,10 +50,10 @@ func (dc *DeltaCalculator) SetExpirationCheckInterval(interval time.Duration) *D
 	return dc
 }
 
-// CountMetric creates a count metric from the difference between the values and
+// GetCumulativeCount creates a count metric from the difference between the values and
 // timestamps of multiple calls.  If this is the first time the name/attributes
 // combination has been seen then the `valid` return value will be false.
-func (dc *DeltaCalculator) CountMetric(name string, attributes map[string]interface{}, val float64, now time.Time) (count telemetry.Count, valid bool) {
+func (dc *DeltaCalculator) GetCumulativeCount(name string, attributes map[string]interface{}, val float64, now time.Time) (count telemetry.Count, valid bool) {
```

Comment:
This is a backwards incompatible change and it does not appear to be warranted. :-1: